Resources/Prototypes/Entities/Objects/Consumable/Food/noodles.yml

```diff
@@ -0,0 +1,104 @@
+# Base
+
+- type: entity
+  parent: BaseItem
+  id: FoodNoodlesBase
+  abstract: true
+  description: Now that's a nic'e pasta!
```

Comment:
```suggestion
  description: Now that's a nice pasta!
```

Reply:
I think this is supposed to imitate an Italian accent, but if you miss that it just looks like a typo, so personally I would still change it.